general/administration/connectivity.md

```diff
@@ -35,3 +35,19 @@ Since client auto-discover would break if this option were configurable, you can
 ## Dynamic Ports
 
 Live TV devices will often use a random UDP port for HD Homerun devices. The server will select an unused port on startup to connect to these tuner devices.
+
+## Base URL
+
+The Base URL setting in the **Networking** page is an advanced setting used to specify the URL prefix that your Jellyfin instance can be accessed at. In effect, it adds this URL fragment to the start of any URL path. For instance, if you have a Jellyfin server at `http://myserver` and access its main page `http://myserver/web/index.html`, setting a Base URL of `/jellyfin` will alter this main page to `http://myserver/jellyfin/web/index.html`. This can be useful if administrators want to access multiple Jellyfin instances under a single domain name, or if the Jellyfin instance lives only at a subpath to another domain with other services listening on `/`.
+
+The entered value on the configuration page will be normalized to include a leading`/` if this is missing.
+
+This setting requires a server restart to change, in order to avoid invalidating existing paths until the administrator is ready to.
+
+There are three main caveats to this setting:
+
+1. When setting a new Base URL (i.e. from `/` to `/baseurl`) or changing a Base URL (i.e. from `/baseurl` to `/newbaseurl`), the Jellyfin web server will automatically handle redirects to avoid displaying users invalid pages. For instance, accessing a server with a Base URL of `/jellyfin` on the `/` path will automatically append the `/jellyfin` Base URL. However, entirely removing a Base URL (i.e. from `/baseurl` to `/`, an empty value in the configuration) will not - all URLs with the old Base URL path will become invalid and throw 404 errors. This should be kept in mind when removing an existing Base URL.
```

Comment:
I don't remember the logic of "redirect on base url change", only when / is changed to something else... am I missing something?

Reply:
It will detect a change. So if you go from `/old` to `/new`, it will realize that `/old` no longer matches what it expects, and redirect back to the main page. It's only full removal that it can't redirect.

Reply:
Whatever you say :)